pnpm-lock.yaml

```diff
@@ -4118,8 +4134,8 @@ packages:
     engines: {node: '>=12'}
     hasBin: true
 
-  esbuild@0.24.2:
-    resolution: {integrity: sha512-+9egpBW8I3CD5XPe0n6BfT5fxLzxrlDzqydF3aviG+9ni1lDC/OvMHcxqEFV0+LANZG5R1bFMWfUrjVsdwxJvA==}
+  esbuild@0.25.0:
```

Comment:
I didn't notice that after those upgrades, electron-vite still uses esbuild 0.21.5 (which is above this entry), I've spotted this only after running `pnpm why -r esbuild`.

At this point, we might as well wait for the automated dependabot PRs since March starts tomorrow.